docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -795,26 +837,211 @@ with either `Ctrl + C` or `kill`.
 
 For more information about K8s port-forward see the [K8s doc](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#port-forward).
 
-### Verify
+## Advanced Setup
 
-Once ready, access the Alluxio CLI from the master Pod and run basic I/O tests.
-```console
-$ kubectl exec -ti alluxio-master-0 /bin/bash
+### Enable Short-circuit Access
```

Comment:
I've moved this to the top of "Advanced Setup" since we have short-circuit enabled by default in our Helm chart.